faker/providers/person/th_TH/__init__.py

```diff
@@ -804,7 +804,7 @@ class Provider(PersonProvider):
         "ถนัดภาษา",
         "ถนัดรบ",
         "ถนัดรักษา",
-        "ถนัดหัดถกรรม",
+        "ถนัดหัตถกรรม",
```

Comment:
I dont see how those two lines are different. Why are they sowing up in the diff?

Reply:
They are two different characters, with a very similar shape (thus very common typo).
One is ด, another with ต.
The only difference is ต has a bend at the top curve of the character, but ด doesn't have the bend.